core/templates/dev/head/dashboard/Dashboard.js

```diff
@@ -16,20 +16,31 @@
  * @fileoverview Controllers for the creator dashboard.
  */
 
+oppia.constant('EXPLORATION_DROPDOWN_STATS', {
+  NEW_FEEDBACK: 'new_feedback',
+  LAST_UPDATES: 'last_updates',
+  TOP_UNRESOLVED_ANSWERS: 'top_unresolved_answers'
+});
+
 oppia.controller('Dashboard', [
   '$scope', '$rootScope', '$window', 'oppiaDatetimeFormatter', 'alertsService',
   'DashboardBackendApiService', 'RatingComputationService',
   'ExplorationCreationService', 'FATAL_ERROR_CODES', 'UrlInterpolationService',
+  'EXPLORATION_DROPDOWN_STATS',
   function(
       $scope, $rootScope, $window, oppiaDatetimeFormatter, alertsService,
       DashboardBackendApiService, RatingComputationService,
-      ExplorationCreationService, FATAL_ERROR_CODES, UrlInterpolationService) {
+      ExplorationCreationService, FATAL_ERROR_CODES, UrlInterpolationService,
+      EXPLORATION_DROPDOWN_STATS) {
     var EXP_PUBLISH_TEXTS = {
       defaultText: (
         'This exploration is private. Publish it to receive statistics.'),
       smText: 'Publish the exploration to receive statistics.'
     };
 
+    $scope.isDropdownOpen = {};
```

Comment:
I agree with @AllanYangZhou. But it seems to me that only one dropdown can be open at a time, is that right? If so, why do you need an object for this? Could you just use a string variable $scope.activeExplorationId or similar (which is null if no dropdown is open)?